README.md

```diff
@@ -5,81 +5,84 @@
-studying the subject can help you avoid some common and very costly mistakes.
-
-The landscape of negotiating for equity—also referred to as stock or ownership—is ridden
-with sometimes mind-numbingly confusing terminology and high-stakes decisions.
-It has also been successfully navigated by many who, too, felt lost when faced with a
-short window in which to learn how to evaluate or deliver a job offer with an equity
-component. The authors and editors of this Guide have navigated the territory of equity
-compensation from the perspective of employees, founders, and lawyers.
-We have created this Guide to help hiring mangaers and employees get a fair shake,
+But it is well understood primarily by tax attorneys, corporate lawyers, and other
+professionals.
+
+Although understanding the technicalities of equity compensation does not guarantee
+fortune will smile upon you as warmly as it did the early hires of Facebook, we do believe
+a good overview can help you make better decisions and avoid some of the most common and
+costly mistakes.
+
+For those daunted by legalities and tax planning, it may help to remember how many feel
+lost and confused when faced with a short window in which to make a key decision, like
+accept a job offer or exercise stock options.
+But millions do get through this process successfully.
+The authors and editors of this Guide have navigated the territory of equity compensation
+from the perspective of employees, founders, and lawyers.
+We have created this Guide to help both employees and those hiring them get a fair shake,
 understand their decisions, and save time and effort.
 
 ### Who is affected?
 
-Equity compensation is a broad term that encompasses several methodologies designed to
+*Equity compensation* is a broad term that encompasses several methodologies designed to
 align the goals of a company with the financial interest of its employees over the long
 term, by giving employees the opportunity to own a piece of their company.
 Companies ranging from two-person startups to Fortune 500 companies utilize equity
 compensation structures to create alignment between business goals and employees.
-As of **2014**, in the United States **7.2%** of all private sector employees (**~8.5M**
+As of **2014** in the United States, **7.2%** of all private sector employees (**~8.5M**
 Americans) and **13.1%** of *all* employees of companies with stock held stock options.
 This number has steadily declined since **2002** levels of **12.3%** and **23.2%** respectively
 ([NCEO analysis](https://www.nceo.org/assets/pdf/articles/GSS-2014-data.pdf) of General
 Social Survey data).
 
-Nearly all approaches to equity compensation are based on the idea that a ownership in a
-company will be more valuable in the future due to growth.
+Generally, equity compensation is based on the idea that ownership in a company will be
+more valuable in the future due to growth.
```

Comment:
We have to be careful about assumptions around whether an audience is are interested in startups or large companies. Ideally covering both.